hawk/src/main/java/com/orhanobut/hawk/Hawk.java

```diff
@@ -178,4 +239,97 @@ public static boolean resetCrypto() {
     public static LogLevel getLogLevel() {
         return logLevel;
     }
+
+    /**
+     * Provides the ability to chain put invocations, for example
+     * <code>Hawk.chain().put("foo", 0).put("bar", 1).done()</code>
+     */
+    public static final class HawkChain {
+
+        private final List<Pair<String, ?>> items;
+        private boolean atomic;
+
+        public HawkChain() {
+            this(4);
```

Comment:
is there any specific reason to use 4? Why not use to default capacity of the array list which is 10 If it is not changed though